components/post_view/embedded_bindings/embedded_binding/embedded_binding.tsx

```diff
@@ -95,15 +91,16 @@ export default class EmbeddedBinding extends React.PureComponent<Props, State> {
                         binding={binding}
                     />,
                 );
-            } else {
-                content.push(
-                    <ButtonBinding
-                        key={binding.location}
-                        post={this.props.post}
-                        binding={binding}
-                    />,
-                );
+                return;
             }
+
+            content.push(
+                <ButtonBinding
+                    key={binding.location}
```

Comment:
It is technically possible, but should not happen. The locations should be properly formed and/or defaulted or filtered both in server and in client. Nevertheless, the impact is low in case it is undefined.